src/utils/index.js

```diff
@@ -59,3 +59,25 @@ export const changed = (props, objectA, objectB) => {
     objectA && objectB && props.some((prop) => objectA[prop] !== objectB[prop])
   );
 };
+
+/**
+ * Replace any parts of a string that match the keys in the toReplace object
+ * @param {string} str The string to check
+ * @returns {string} The string with or without replaced values
+ */
+const toReplace = {
+  '&lt;': '<',
+  '&gt;': '>',
+};
+
+export const replaceMatches = (str) => {
+  if (str && str.length > 0) {
```

Comment:
Curious: what is this conditional for here? I sort of understand the zero-length string case but not the case where `bool(str)` would evaluate to False - when does that occur?

Reply:
If you don't have that check you'd get this error: `TypeError: Cannot read property 'length' of undefined`. I've changed it now to this `str?.length > 0` which does the same thing: first checks if there's a string, and if so, proceed to find the length.